code/modules/language/canilunzt.dm

```diff
@@ -0,0 +1,15 @@
+/datum/language/canilunzt
+	name = "Canilunzt"
+	desc = "The common language of Ethari, composed of low growls and yelps."
+	speech_verb = "growls"
+	ask_verb = "growls"
+	exclaim_verb = "roars"
+	key = "7"
+	flags = TONGUELESS_SPEECH
```

Comment:
erm, this is a wierdly named flag - what does it actually do?

Reply:
Means you can speak it without a tongue. You still have issues with pronouncing most letters, but you can speak the language.

Reply:
seems goofy